cli/services/helm.go

```diff
@@ -63,7 +76,83 @@ func (h *helm3_1_1) DeleteChart(chart string) error {
 	return nil
 }
 
-func (h *helm3_1_1) InstallChart(name string, chart string, version string) error {
+func (h *helm3_1) Init() error {
+	return nil
+}
+
+func (h *helm3_1) InstallChart(name string, chart string, version string) error {
+	args := []string{
+		"install", name, chart, "--version", version,
+	}
+
+	output, err := helmExecute(args...)
+	if err != nil {
+		return err
+	}
+	log.WithFields(log.Fields{
+		"chart":   chart,
+		"name":    name,
+		"output":  output,
+		"version": version,
+	}).Debug("Chart installed")
+
+	return nil
+}
+
+type helm2_16 struct {
+	helm
+}
+
+func (h *helm2_16) AddRepo(repo string, URL string) error {
+	// use chart as application name
+	args := []string{
+		"repo", "add", repo, URL,
+	}
+
+	output, err := helmExecute(args...)
+	if err != nil {
+		return err
+	}
+
+	log.WithFields(log.Fields{
+		"output": output,
+		"name":   repo,
+		"url":    URL,
+	}).Debug(repo + " Helm charts added")
+	return nil
+}
+
+func (h *helm2_16) DeleteChart(chart string) error {
+	args := []string{
+		"delete", "--purge", chart,
+	}
+
+	output, err := helmExecute(args...)
+	if err != nil {
+		return err
+	}
+
+	log.WithFields(log.Fields{
+		"output": output,
+		"chart":  chart,
+	}).Debug("Chart deleted")
+	return nil
+}
+
+func (h *helm2_16) Init() error {
+	args := []string{
+		"init", "--namespace", "default",
```

Comment:
This command is failing because the underlying image is not pulled from gcr.io:

```
$ kubectl get pods --namespace default
NAME                             READY   STATUS         RESTARTS   AGE
tiller-deploy-7dffd9c7c7-6s8vn   0/1     ErrImagePull   0          21s
```

Reply:
Solved using Kind instead of minikube